docs/workflow/requirements/macos-requirements.md

```diff
@@ -23,47 +23,14 @@ Install the following packages:
 - cmake 3.15.5 or newer
 - autoconf
 - automake
+- icu4c
 - libtool
+- openssl 1.1 or newer
```

Comment:
Does newer work? I may have incorrectly thought we were stuck on 1.1. /cc @wfurt @janvorli

Reply:
We used to be stuck on 1.0.2; now we work anything 1.0.1-1.1.1 (though only 1.1.1 is still in support)

Reply:
Yes it works. We no longer do dynamic linking. We just use it to build and we link at runtime against whatever is installed on the running machine. cc: @bartonjs

Reply:
I removed the *or newer* in f0f57416acc1cf78d2e96f2ed50492d1cb719e06. Let's be conservative about version 1.1.